modules/vault-elb/main.tf

```
@@ -24,6 +24,17 @@ resource "aws_elb" "vault" {
   security_groups = [aws_security_group.vault.id]
   subnets         = var.subnet_ids
 
+  # optional access_logs creation  
+  dynamic "access_logs" {
+    for_each = length(keys(var.access_logs)) == 0 ? [] : [var.access_logs]
```

```suggestion
    for_each = var.access_logs == null ? [] : ["once"]
```

See the comment below for why I'm checking against `null`.